lib/manageiq/api/common/rbac/access.rb

```diff
@@ -0,0 +1,72 @@
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class Access
+          attr_reader :acl
+          DEFAULT_LIMIT = 500
+          def initialize(resource, verb)
+            @resource = resource
+            @verb     = verb
+            @regexp   = Regexp.new(":(#{@resource}|\\*):(#{@verb}|\\*)")
```

Comment:
This feels like the wrong way to build this regex, particularly if resource or verb have regexp characters in them (particularly a `.` or a paren)...I think this is more accurate:

```ruby
@regexp   = /:(#{Regexp.escape(@resource)}|\\*):(#{Regexp.escape(@verb)}|\\*)/
```